heaps/min_heap.py

```diff
@@ -57,19 +77,48 @@ def heap_up(self, index):
             property is reestablished.
             
             This could be **very** helpful for the add method.
-            Time complexity: ?
-            Space complexity: ?
+            Time complexity: O(log n)
+            Space complexity: O(1)
```

Comment:
✨ Nice, however the `heap_up` operation is O(log n) because of the recursive call stack